osu.Game/Online/API/Requests/Responses/APIScoreInfo.cs

```diff
@@ -15,9 +15,68 @@
 
 namespace osu.Game.Online.API.Requests.Responses
 {
-    public class APILegacyScoreInfo
+    public class APIScoreInfo : IScoreInfo
     {
-        public ScoreInfo CreateScoreInfo(RulesetStore rulesets)
+        [JsonProperty(@"score")]
+        public long TotalScore { get; set; }
+
+        [JsonProperty(@"max_combo")]
+        public int MaxCombo { get; set; }
+
+        [JsonProperty(@"user")]
+        public User User { get; set; }
+
+        public bool HasReplay { get; set; }
+
+        [JsonProperty(@"id")]
+        public long OnlineID { get; set; }
+
+        [JsonProperty(@"replay")]
+        public bool Replay { get; set; }
```

Comment:
I think this may be the same thing as `HasReplay`? Hard to tell because docs are... scarce, but maybe this could - or even should - implement the interface field?